packages/cashier/src/pages/account-transfer/account-transfer-locked/__tests__/account-transfer-locked.spec.tsx

```diff
@@ -41,10 +41,12 @@ describe('AccountTransferLocked', () => {
             {
```

Comment:
there are no types for items.

Reply:
I guess we don't need types for these constant, because by default ts can detect the types

Reply:
Ts can infer it. but I added type to `onClick`

Reply:
I checked and we don't need type for onClick. removed it